protocol/app/msgs/unsupported_msgs.go

```diff
@@ -18,6 +20,41 @@ var (
 		"/cosmos.gov.v1beta1.MsgSubmitProposal":         &govbeta.MsgSubmitProposal{},
 		"/cosmos.gov.v1beta1.MsgSubmitProposalResponse": nil,
 
+		// cosmwasm
+		"/cosmwasm.wasm.v1.AcceptedMessageKeysFilter":                  nil,
```

Comment:
Could we add brief reasoning for why these are unsupported:

From rough look:
- 24 ~ 26: `authz` related messages 
- 27: Deprecated
- 28 ~30: `authz` related messages